sts-mock/src/token/configService/configService.ts

```diff
@@ -12,9 +12,9 @@ export interface IGetConfig<T> {
 
 export class ConfigService implements IGetConfig<Config> {
```

Comment:
Once you've checked if the URL envVars are there, you could also do something like:

```
try {
      new URL(env.STS_MOCK_BASE_URL);
    } catch {
      return errorResult({
        errorMessage: "STS_JWKS_ENDPOINT is not a URL",
        errorCategory: "SERVER_ERROR",
      });
    }
```

Reply:
That way if it's not a valid URL we're not bothering executing the rest of the lambda since it won't work anyway

Reply:
We could add this additional check but I don't think it's worth the time considering that this is a mock that will be used by developers. Those env var values are defined in the SAM template and we shouldn't be "messing" with them. I think there are so many things we could check but sometimes checking for everything doesn't save time. What do you think? 🤔

Reply:
I think let's just get this in as-is - it's a mock asset and if we've got an incorrect URL the request to the well-known endpoint will fail with an error. If it becomes a regular issue (e.g. if it's very easy to make mistakes when deploying dev stacks) let's add it.